tdd_own_test_cases.js

```diff
@@ -0,0 +1,32 @@
+function isAllCaps(text) {
+	return text === text.toUpperCase();
+}
+
+function countVowels(text) {
+	const vowels = ["a", "e", "i", "o", "u"];
+	const vowelsOfText = text
+		.split("")
+		.filter((letter) => vowels.includes(letter));
+	return vowelsOfText.length;
+}
+
+function mergeAndSortArrays(arr1, arr2) {
+	const mergedArray = [...arr1, ...arr2];
```

Comment:
you could have only had a return statement if you returned [...arr1, ...arr2].sort()